src/actions/acquire_stepped_freq_tdomain_iq.py

```diff
@@ -16,8 +16,7 @@
 # - Markdown reference: https://commonmark.org/help/
 # - SCOS Markdown Editor: https://ntia.github.io/scos-md-editor/
 #
-r"""Capture time-domain IQ samples at {nfcs} frequencies between
-{f_low_edge:.2f} and {f_high_edge:.2f} MHz.
+r"""Capture time-domain IQ samples at the following {nfcs} frequencies (MHz): {frequencies}.
```

Comment:
A few comments:

- This description will become comically long if `self.fcs` is long since it will describe each and every frequency in the list, e.g.

```Capture time-domain IQ samples at the following 1000 frequencies (MHz): 1.000, 2.000, 3.000, 4.000, 5.000, ......```

I might suggest some logic to set an upper cap as to how many frequencies should be listed before turning it into a range. Maybe Todd has insight/disagreement on me here?

- I'm not sure if this format is the best option, but I'm not the intended audience so I'm happy to be vetoed here. My suggestion also aggreviates my first bullet's problem, so I'm open to conversation as to what the best idea here is.

Would something like this be more appealing or clear? My concern is that pulling out the unit and just providing a list of floats may be undesirable.

```
Change description to: 
Capture time-domain IQ samples at the following {nfcs} frequencies: {frequencies}.

Change "frequencies" to:
"frequencies": ", ".join(["{:.2f} MHz".format(fc / 1e6) for fc in self.fcs])

Before:
Capture time-domain IQ samples at the following 4 frequencies (MHz): 1.000, 2.000, 3.000, 4.000

After:
Capture time-domain IQ samples at the following 4 frequencies: 1.000 MHz, 2.000 MHz, 3.000 MHz, 4.000 MHz
```

Reply:
For this action, I do not think the frequencies are necessarily related to each other, so I think the user would need to know exactly which frequencies they are before scheduling the acquisition. A different action with linearly spaced frequencies, possibly called "acquire_swept_freq_tdomain_iq", might be better served with a range in the description. In the case of exceedingly long lists of frequencies, a separate action may be better served to describe them or possibly a substitute for the description.

In the same vein, adding the unit after each frequency helps denote that these aren't a linear range, which is personally a little easier to read to see which frequencies are actually used.
`1.250, 1.457, 1.684, 1.698, 2.145, 2.541`
vs
`1.250 MHz, 1.457 MHz, 1.684 MHz, 1.698 MHz, 2.145 MHz, 2.541 MHz`

Reply:
I added MHz after each frequency per @ToddSchumann suggestion.